sonaranalyzer-dotnet/src/SonarAnalyzer.Common/SonarAnalyzer.Common.csproj

```diff
@@ -19,6 +19,10 @@
     <Compile Include="Protobuf\AnalyzerReport.cs">
       <SonarQubeExclude>true</SonarQubeExclude>
     </Compile>
+    <Compile Remove="Protobuf\Ucfg.cs" />
+    <Compile Include="Protobuf\Ucfg.cs">
```

Comment:
Minor: it should be possible to use Item [Update](https://docs.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild) here.